specs/packagemanager/PackageManagement.md

```diff
@@ -398,6 +400,24 @@ Is\*Provisioned\*() methods determine if the target is provisioned.
 
 These methods require administrative privileges.
 
+## 3.13. PackageValidator
+
+This API allows callers to verify that packages being processed by PackageDeploymentManager.AddPackageAsync
+and PackageDeploymentManager.StagePackageAsync match what are expected from their URI.
+
+When adding or staging a package from an external source such as HTTP URI or uncontrolled file location,
+an attacker can potentially intercept and tamper with the package data being read, causing a malicious
+package to be installed instead of the expected one.  Verifying the identity and signature of target
+packages helps ensure that such tampering has not happened.
+
+The following PackageValidators are provided and available for use directly:
+* PackageFamilyNameValidator: Validates that the package has the expected package family name.
+* PackageMinimumVersionValidator: Validates that the package has at least the expected minimum version number.
+* PackageCertificateEkuValidator: Validates that the certificate used to sign the package contains the expected Extended Key Usage (EKU) value.
+
+Custom validators can also be implemented using the IPackageValidator interface, which allows verifying
```

Comment:
SUGGESTION: Phrasing tweak:

```
Custom validators can be implemented using the `IPackageValidator` interface. This can verify any part of the package’s [footprint data](https://learn.microsoft.com/windows/win32/api/appxpackaging/ne-appxpackaging-appx_bundle_footprint_file_type) (manifest, block map, and digital signature).
```